Initialize engine version: 5.5.1f1 (88d00a7498cd)
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: AMD Radeon (TM) R7 370 Series Graphics (ID=0x6811)
    Vendor:   ATI
    VRAM:     3072 MB
    Driver:   15.300.1025.1001
Begin MonoManager ReloadAssembly
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.dll (this message is harmless)
Loading C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.dll into Unity Child Domain
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\Assembly-CSharp.dll (this message is harmless)
Loading C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\Assembly-CSharp.dll into Unity Child Domain
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.UI.dll (this message is harmless)
Loading C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.UI.dll into Unity Child Domain
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.Networking.dll (this message is harmless)
Loading C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.Networking.dll into Unity Child Domain
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.PlaymodeTestsRunner.dll (this message is harmless)
Loading C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\UnityEngine.PlaymodeTestsRunner.dll into Unity Child Domain
- Completed reload, in  1.580 seconds
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\System.Core.dll (this message is harmless)
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\System.dll (this message is harmless)
<RI> Initializing input.
<RI> Input initialized.
desktop: 1920x1080 60Hz; virtual: 3840x1080 at 0,0
<RI> Initialized touch support.
UnloadTime: 1.228169 ms
Unloading 5 Unused Serialized files (Serialized files now loaded: 0)
UnloadTime: 5.511084 ms

Unloading 3 unused Assets to reduce memory usage. Loaded Objects now: 746.
Total: 0.429123 ms (FindLiveObjects: 0.032960 ms CreateObjectMapping: 0.030080 ms MarkObjects: 0.359362 ms  DeleteObjects: 0.005760 ms)

9532
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

127.0.0.1
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\System.Configuration.dll (this message is harmless)
Platform assembly: C:\Users\Pablo Alejos\Documents\GitHub\Machine-Learning-GameDev\RedesNeuronales\AI_Data\Managed\System.Xml.dll (this message is harmless)
SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

SocketException :System.Net.Sockets.SocketException: Se produjo un error durante el intento de conexión ya que la parte conectada no respondió adecuadamente tras un periodo de tiempo, o bien se produjo un error en la conexión establecida ya que el host conectado no ha podido responder.

  at System.Net.Sockets.Socket.Receive (System.Byte[] buffer) [0x00000] in <filename unknown>:0 
  at SocketController.receiveMessage () [0x00000] in <filename unknown>:0 
  at SocketController.SendMessage () [0x00000] in <filename unknown>:0 
 
(Filename: C:/buildslave/unity/build/artifacts/generated/common/runtime/DebugBindings.gen.cpp Line: 51)

InvalidOperationException: Process has exited, so the requested information is not available.
  at System.Diagnostics.Process.get_ProcessName () [0x00000] in <filename unknown>:0 
  at System.Diagnostics.Process.ToString () [0x00000] in <filename unknown>:0 
  at UnityEngine.Logger.GetString (System.Object message) [0x00000] in <filename unknown>:0 
  at UnityEngine.Logger.Log (LogType logType, System.Object message) [0x00000] in <filename unknown>:0 
  at UnityEngine.Debug.Log (System.Object message) [0x00000] in <filename unknown>:0 
  at SocketController.KillProcess () [0x00000] in <filename unknown>:0 
  at SocketController.OnApplicationQuit () [0x00000] in <filename unknown>:0 
 
(Filename:  Line: -1)

Setting up 2 worker threads for Enlighten.
  Thread -> id: 2ab8 -> priority: 1 
  Thread -> id: 25e8 -> priority: 1 
InvalidOperationException: Process has exited, so the requested information is not available.
  at System.Diagnostics.Process.get_ProcessName () [0x00000] in <filename unknown>:0 
  at System.Diagnostics.Process.ToString () [0x00000] in <filename unknown>:0 
  at UnityEngine.Logger.GetString (System.Object message) [0x00000] in <filename unknown>:0 
  at UnityEngine.Logger.Log (LogType logType, System.Object message) [0x00000] in <filename unknown>:0 
  at UnityEngine.Debug.Log (System.Object message) [0x00000] in <filename unknown>:0 
  at SocketController.KillProcess () [0x00000] in <filename unknown>:0 
  at SocketController.OnDestroy () [0x00000] in <filename unknown>:0 
 
(Filename:  Line: -1)

